frontend/public/config.yml

```
@@ -465,7 +465,63 @@ collections:
               { label: 'Obrázok', name: 'image', widget: 'image' },
             ]
           }
-      
+        #tabor
+            - { label: 'Tábor nadpis', name: 'tabor_header', widget: 'string' }
+            - { label: 'Tábor popis', name: 'tabor_body', widget: 'markdown' }
+            - { label: 'Tábor obrázok', name: 'tabor_image', widget: 'image' }
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)